cli/lib/kontena/plugin_manager.rb

```diff
@@ -17,10 +19,17 @@ def load_plugins
           plugin = File.join(spec.gem_dir, require_path, 'kontena_cli_plugin.rb')
           if File.exist?(plugin) && !@plugins.find{ |p| p.name == spec.name }
             begin
-              load(plugin)
-              @plugins << spec
+              kontena_cli = spec.runtime_dependencies.find{ |d| d.name == 'kontena-cli' }
+              if !kontena_cli.match?('kontena-cli', MIN_CLI_VERSION)
```

Comment:
minor issue: some comment for this might be good, as the negation makes it a bit mind-bender. :)